Automattic/jetpack
# Unit Tests: Update coverage annotations

Official PHPUnit documentation suggests specific use of `@covers` annotations in tests: https://phpunit.readthedocs.io/en/9.5/annotations.html#covers

This PR updates all invalid annotations, and update valid ones to match PHPUnit recommendations. 

These changes improve code coverage for +0.33% or ~300 LOC.

#### Jetpack product discussion


n/a
#### Does this pull request change what data or activity we track or use?


n/a
#### Testing instructions:






* Check CI "Coverage" job, and make sure there are no PHPUnit warnings generated regarding invalid `@covers` usage
